Formula/s/strongswan.rb

```diff
@@ -32,10 +32,10 @@ class Strongswan < Formula
 
   depends_on "openssl@3"
 
+  uses_from_macos "curl" # curl is used from macOS
```

Comment:
No need to add a comment here
```suggestion
  uses_from_macos "curl"
```